sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClientBuilder.java

```diff
@@ -105,9 +105,7 @@ public ConfigurationClientBuilder() {
         policies = new ArrayList<>();
         httpLogOptions = new HttpLogOptions();
 
-        Map<String, String> properties = CoreUtils.getProperties(APP_CONFIG_PROPERTIES);
-        clientName = properties.getOrDefault(NAME, "UnknownName");
-        clientVersion = properties.getOrDefault(VERSION, "UnknownVersion");
+        properties = CoreUtils.getProperties(APP_CONFIG_PROPERTIES);
```

Comment:
This is changing previous behavior. With the changes to get the map and only calling `.get` the default will be `null` instead of `UnknownName` and `UnknownVersion`. You'll want to update the `get` calls to `getOrDefault` when constructing the `UserAgentPolicy`.